features/step_definitions/ada-redemption-steps.js

```diff
@@ -221,9 +221,9 @@ When(
         '.AdaRedemptionForm_component .pass-phrase input',
         word
       );
-      await this.client.waitForVisible(`//li[contains(text(), '${word}')]`);
-      await this.waitAndClick(`//li[contains(text(), '${word}')]`);
-      await this.client.waitForVisible(`//span[contains(text(), '${word}')]`);
+      await this.client.waitForVisible(`//li[text()="${word}"]`);
+      await this.waitAndClick(`//li[text()="${word}"]`);
```

Comment:
`this.waitAndClick` already does `waitForVisible` so you can remove those extra checks